contrib/asff.tpl

```diff
@@ -82,7 +82,7 @@
         {
             "SchemaVersion": "2018-10-08",
             "Id": "{{ $target }}/{{ .ID }}",
-            "ProductArn": "arn:aws:securityhub:{{ env "AWS_REGION" }}::product/aquasecurity/aquasecurity",
+            "ProductArn": "arn:aws:securityhub:{{ env "AWS_REGION" }}:{{ env "AWS_ACCOUNT_ID" }}:product/aquasecurity/trivy",
```

Comment:
I'm not familiar with ASFF, but don't we need to keep it aquasecurity/aquasecurity?
https://github.com/aws-samples/aws-security-hub-scan-with-trivy/blob/49ca23f242603dd1b8043f22b670c5323d92b6da/sechub_parser.py#L74

Reply:
I remember there is permission importing findings to Security Hub. Isn't the ProductArn relevant?

Reply:
@knqyf263 I took a loot at the documentation
https://docs.aws.amazon.com/service-authorization/latest/reference/list_awssecurityhub.html

and found next definition of resource type product:
```
arn:${Partition}:securityhub:${Region}:${Account}:product/${Company}/${ProductId}
```

Reply:
@AndrewCharlesHay After you changed ProductArn, did you confirm the ASFF file could be imported into Security Hub?

Reply:
Yeah I have local version of this file with just the arn changed and I am able to upload to Security Hub